src/test/java/racingcar/domain/CarDomainsTest.java

```diff
@@ -0,0 +1,71 @@
+package racingcar.domain;
+
+import static camp.nextstep.edu.missionutils.test.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static racingcar.common.constant.Constant.MOVE_FORWARD_RANGE;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class CarDomainsTest {
+    private CarDomains carDomains;
+
+    @BeforeEach
+    public void setUp() {
+        CarDomain car1 = CarDomain.create("Car1", 0);
+        CarDomain car2 = CarDomain.create("Car2", 0);
+        carDomains = CarDomains.create(Arrays.asList(car1, car2));
+    }
+
+    @Test
+    @DisplayName("자동차_초기화_테스트")
+    public void testInitialCars() {
+        assertSimpleTest(() -> {
+            List<CarDomain> cars = carDomains.getCarDomainList();
+            assertNotNull(cars);
+            assertEquals(2, cars.size());
+            assertEquals("Car1", cars.get(0).getName());
+            assertEquals("Car2", cars.get(1).getName());
+            assertEquals(0, cars.get(0).getDistance());
+            assertEquals(0, cars.get(1).getDistance());
```

Comment:
AssertJ 스타일로 변경하는것은 어떻게 생각하세요?

AssertJ를 사용하면

```suggestion
            assertThat(cars)
                .isNotNull()
                .hasSize(2)
                .extracting("name", "distance")
                .containsExactly(
                    tuple("Car1", 0),
                    tuple("Car2", 0)
                );
```
처럼 나타낼 수 있어요.

 메서드 체이닝으로 가독성이 좋고, 테스트 실패 시 더 명확한 에러 메시지를 제공하며, 특히 컬렉션을 검증할 때 유용한 기능들이 많아서 선호하는 편이에요.

Reply:
와 이건 진짜... 굳이네요...! 너무 감사드립니다!!

Reply:
영어는 개인적인 취향이라 영어로 적었습니다. 그런데 대문자 소문자에 대한 지표가 좀 애매하더라구요? 어디는 대문자 사용하라고 하고 어디는 소문자 사용하라고 해서 일단 앞으로는 소문자로 사용하려고 합니다!

좋은 말씀 감사합니다! 좀 더 생각해볼 수 있는 시간이 되었습니다!